### PR TITLE
Jumpstart ag - ADX dashaboards data emulator changes

### PR DESCRIPTION
**Pull request by chintalavr**
This release includes folder restructure to keep Azure Data Explorer dashboard templates and data emulator executable to run manually.

### CODE REVIEW

File: azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1 [hunk 1]
```diff
@@ -260,18 +260,31 @@ if ($httpResponse.StatusCode -ne 200){
...
+        Write-Host "[$(Get-Date -Format t)] ERROR: Unable to find Azure Data Explorer endpoint from the cluster resource in the resource group."
+    }
 }
 
+# Download DataEmulator.zip into Agora folder and unzip
+$emulatorPath = "$dataEmulatorDir\DataEmulator.zip"
+Invoke-WebRequest -Method Get -Uri "$templateBaseUrl/artifacts/data_emulator/DataEmulator.zip" -OutFile $emulatorPath
+
+# Unzip DataEmulator.zip to copy DataEmulator exe and config file to generate sample data for dashboards
+if ([System.IO.File]::Exists($emulatorPath)) {
```
Comment (dkirby-ms) [inline azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1:284]: consider Test-Path instead of the .NET call here for future use

**Review by dkirby-ms — CHANGES_REQUESTED** [review]

File: azure_jumpstart_arcbox/artifacts/statys.json [hunk 2]
```diff
@@ -0,0 +1,162 @@
+{
```
[the remark this answers is not in the document]
Comment (chintalavr) [reply azure_jumpstart_arcbox/artifacts/statys.json:1]: My bad, this is not supposed to be committed. :( Deleting.

File: azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1 (hunk 1 above)
Comment (chintalavr) [reply azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1:284]: Good advise, will change in this release. :)

File: azure_jumpstart_arcbox/artifacts/statys.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (chintalavr) [reply azure_jumpstart_arcbox/artifacts/statys.json]: Deleted

File: azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1 (hunk 1 above)
Comment (chintalavr) [reply azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1]: Updated

**Review by dkirby-ms — APPROVED** [review]

**Pull request merged**